.eslintrc

```diff
@@ -1,4 +1,159 @@
 {
-  "plugins": ["@theforeman/foreman"],
-  "extends": "plugin:@theforeman/foreman/core"
+  "plugins": ["@theforeman/foreman", "spellcheck"],
+  "extends": "plugin:@theforeman/foreman/core",
+  "rules": {
+    "spellcheck/spell-checker": [
+      "warn",
+      {
+        "comments": false,
+        "identifiers": false,
+        "strings": true,
+        "lang": "en_US",
+        "ignoreRequire": false,
+        "skipWords": [
+          "2xl",
+          "altavista",
+          "ampm",
+          "auditable",
+          "bool",
+          "bootable",
+          "btns",
+          "candlepin",
+          "centos",
+          "checkbox",
+          "clearbutton",
+          "clearfix",
+          "consts",
+          "csrf",
+          "csv",
+          "datastore",
+          "datastores",
+          "datepicker",
+          "datetime",
+          "datetimepicker",
+          "dbde",
+          "debian",
+          "decrement",
```

Comment:
Odd that this is needed. It's a legitimate English word. Upstream bug report?

Reply:
Will do 👍

Reply:
Opened https://github.com/aotaduy/eslint-plugin-spellcheck/issues/67